data/data/agent/files/usr/local/bin/load-config-iso.sh

```diff
@@ -64,6 +65,9 @@ copy_archive_contents() {
 
     echo "Successfully copied contents of ${AGENT_CONFIG_ARCHIVE_FILE} on ${devname}"
 
+    # Update core password with one from config-image
+    usermod --password "$(cat ${PASSWORD_HASH})" core
```

Comment:
Won't it affect only the bootstrap step? I mean, after rebooting and start the installation it won't be possible to login using that password?
Also, kubeadmin-password is quite long, so it isn't ideal for connecting through console.
Can we maybe make this behaviour optional some how? I.e. some env var or file to override the password with a custom one. Or just disable the update of the password.

Reply:
So I guess I don't understand the questions. This fix brings the use of the config-image to parity with the normal installation flow. The config-image was missing the user that gets created when doing an installation and this fixes it. 

Yes this allows console login with the `kubeadmin-password` during bootstrap only, which can very useful if the hosts are not booting properly and network errors prevent ssh login.  Is there a reason why we'd want to disable this?

Reply:
I talked with Daniel at the same time and this was his response. Still thinking about the best way to handle this...

> So the issue with that change is that we won’t be able to override the core user pass. I mean, in the appliance we have an option to specify a custom ‘userCorePass’: https://github.com/openshift/appliance/blob/master/docs/user-guide.md#set-appliance-config
> So this is useful as some partners want to connect to console using some easy predefined password. Especially for the bootstrap step - to test the machine, etc. Not sure it’s crucial for after rebooting the machine to the ocp installation step.
> Any way, it would be great if you could just add some condition before setting the password. I.e. an env var, a file or any indication. Just so we could override it for the appliance flow.